src/yunohost/certificate.py

```diff
@@ -731,13 +731,7 @@ def _dns_ip_match_public_ip(public_ip, domain):
 
 def _domain_is_accessible_through_HTTP(ip, domain):
     try:
-        # Check HTTP reachability
         requests.head("http://" + ip, headers={"Host": domain})
```

Comment:
As discussed on the chat, even requesting `/yunohost/admin` or `/yunohost/api` doesn't give a clear indication. Properly diagnosing the status/configuration of a domain seems to be a complicated problem :stuck_out_tongue:. Let's agree that these are some minimal checks for now, and can be refined later.